src/app/dashboard/whitelist/style.scss

```diff
@@ -0,0 +1,75 @@
+$main-font: 'Overpass', sans-serif;
+
+.modalHeight {
+  height: 600px
+}
+
+%modal-text {
+  font-family: $main-font;
```

Comment:
do we really have to specify font family for each css block?
we use only Overpass everywhere

Reply:
i removed all font family stuff in here. it should all be inherited from polymath ui